Roslyn.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>  
+<RunSettings>  
+  <RunConfiguration>  
+    <TestAdaptersPaths>%userprofile%\.nuget\packages\xunit.runner.visualstudio\2.2.0-beta4-build1194</TestAdaptersPaths>  
```

Comment:
Not sure... @tmeschter do you figure the nuget APIs will work correctly for CPS based projects with <PackageReference>?  (with regard to being able to evaluate versions specified using MSBuild properties)